packages/sync-repo-settings/test/test.ts

```diff
@@ -104,18 +108,11 @@ async function receive(org: string, repo: string, cronOrg?: string) {
 
 describe('Sync repo settings', () => {
   beforeEach(() => {
-    probot = new Probot({
-      // eslint-disable-next-line node/no-extraneous-require
-      Octokit: require('@octokit/rest').Octokit,
+    probot = createProbot({
+      githubToken: 'abc123',
+      Octokit: TestingOctokit as any,
```

Comment:
Why does this need to be `any`?

Reply:
I'll be honest, I took this part from @bcoe's upgrade of release-please. But, I think this is because this way, we can test using a separate instance of Octokit, as opposed to using the one that comes with Probot.

Reply:
we might be able to make it an `instanceof Octokit`, I believe I tried this and it was unhappy though.